tests/bake.go

```diff
@@ -887,6 +888,56 @@ target "def" {
 	require.Len(t, md.BuildWarnings, 3, string(dt))
 }
 
+func testBakeMultiPlatform(t *testing.T, sb integration.Sandbox) {
+	registry, err := sb.NewRegistry()
+	if errors.Is(err, integration.ErrRequirements) {
+		t.Skip(err.Error())
+	}
+	require.NoError(t, err)
+	target := registry + "/buildx/registry:latest"
+
+	dockerfile := []byte(`
+	FROM --platform=$BUILDPLATFORM busybox:latest AS base
+	COPY foo /etc/foo
+	RUN cp /etc/foo /etc/bar
+
+	FROM scratch
+	COPY --from=base /etc/bar /bar
+	`)
+	bakefile := []byte(`
+	target "default" {
+	platforms = ["linux/amd64", "linux/arm64"]
+	}
+	`)
+	dir := tmpdir(
+		t,
+		fstest.CreateFile("docker-bake.hcl", bakefile, 0600),
+		fstest.CreateFile("Dockerfile", dockerfile, 0600),
+		fstest.CreateFile("foo", []byte("foo"), 0600),
+	)
+
+	cmd := buildxCmd(sb, withDir(dir), withArgs("bake"), withArgs("--set", fmt.Sprintf("*.output=type=image,name=%s,push=true", target)))
+	out, err := cmd.CombinedOutput()
+
+	if !isMobyWorker(sb) {
```

Comment:
@crazy-max how does this check detect whether the docker daemon supports multi-arch? Having a quick look; `isMobyWorker` and `isMobyContainerdSnapWorker` look identical, but if docker is running with the containerd image-store enabled, it supports multi-arch? https://github.com/docker/buildx/blob/56cb197c0a186e4382eba26537af238e19fe880c/tests/integration.go#L97-L105

Should we have a `supportsMultiArch` check instead?

Reply:
OH! Nevermind; `hasFeature` vs `!hasFeature` 🙈 I missed the `!` 🙈 😂 


Still, maybe a `supportsMultiArch` or `hasMultiarchSupport` could make sense (more descriptive?)

Reply:
This is not just multiarch but also cache and oci exporters that are supported with containerd snap: https://github.com/docker/buildx/blob/aa35c954f3f1a346ef71a1e07f942d2a46740b1f/driver/docker/driver.go#L92-L95